doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,171 @@
+
+# User Guide
+
+The goal of this library is to provide a general framework to implement complex 
+data analysis algorithms with Exasol. This framework provides certain features 
+to users in which they are able to run their implementations.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Setup](#setup)
+- [Usage](#usage)
+- [Implementation of Algorithms](#implementation-of-algorithms)
+
+
+
+## Getting Started
+- Exasol DB
+  - The Exasol cluster must already be running with version 7.1 or later.
+  - DB connection information and credentials are needed.
+- TODO: Connection
+- TODO: Algorithm implementation
```

Comment:
obsolete

Reply:
Regarding the Algorithm implementation, I had the same impression.
For the connection: see next push.